examples/gno.land/p/demo/uassert/uassert.gno

```diff
@@ -0,0 +1,261 @@
+// uassert is an adapted lighter version of https://github.com/stretchr/testify/assert.
+package uassert
+
+import (
+	"std"
+	"strconv"
+	"strings"
+)
+
+// NoError asserts that a function returned no error (i.e. `nil`).
+func NoError(t TestingT, err error, msgs ...string) bool {
+	t.Helper()
+	if err != nil {
+		return fail(t, msgs, "unexpected error: %s", err.Error())
+	}
+	return true
+}
+
+// Error asserts that a function returned an error (i.e. not `nil`).
+func Error(t TestingT, err error, msgs ...string) bool {
+	t.Helper()
+	if err == nil {
+		return fail(t, msgs, "an error is expected but got nil")
+	}
+	return true
+}
+
+// ErrorContains asserts that a function returned an error (i.e. not `nil`)
+// and that the error contains the specified substring.
+func ErrorContains(t TestingT, err error, contains string, msgs ...string) bool {
+	t.Helper()
+
+	if !Error(t, err, msgs...) {
+		return false
+	}
+
+	actual := err.Error()
+	if !strings.Contains(actual, contains) {
+		return fail(t, msgs, "error %q does not contain %q", actual, contains)
+	}
+
+	return true
+}
+
+// True asserts that the specified value is true.
+func True(t TestingT, value bool, msgs ...string) bool {
+	t.Helper()
+	if !value {
+		return fail(t, msgs, "should be true")
+	}
+	return true
+}
+
+// False asserts that the specified value is false.
+func False(t TestingT, value bool, msgs ...string) bool {
+	t.Helper()
+	if value {
+		return fail(t, msgs, "should be false")
+	}
+	return true
+}
+
+// ErrorIs asserts the given error matches the target error
+func ErrorIs(t TestingT, err, target error, msgs ...string) bool {
```

Comment:
```suggestion
func IsError(t TestingT, err, target error, msgs ...string) bool {
```

Reply:
`ErrorIs` is the wording in the assert package, which this is trying to mimic
https://pkg.go.dev/github.com/stretchr/testify/assert#ErrorIs